app/components/modules/TopRightMenu.jsx

```diff
@@ -78,7 +77,7 @@ function TopRightMenu({username, showLogin, logout, loggedIn, vertical, navigate
         return (
             <ul className={mcn + mcl}>
                 {!vertical && <li className="Header__search"><a href="/static/search.html" title="Search"><Icon name="search" /></a></li>}
-                <li className={lcn}><LoadingIndicator type="circle" inline /></li>
+                <li className={lcn} style={{paddingTop: 0, paddingBottom: 0}}><LoadingIndicator type="circle" inline /></li>
```

Comment:
Please try to avoid inline styling - we are planning to disallow it in CSP headers. If allowed, inline styles can be injection point for malicious code. Unfortunately we can't do it right now - inline styles are used by foundation library, so we need to either switch to new css framework or fix foundation.